wfl/Controllers/RestaurantsController.cs

```diff
@@ -42,7 +42,11 @@ public RestaurantsController(IDBManager dbManager)
         [HttpGet]
         public async Task<IHttpActionResult> GetAsync()
         {
-            return Ok(await DBManager.GetAllRestaurantsAsync());
+            RestaurantCollection restaurantCollection = new RestaurantCollection()
```

Comment:
I'm going to merge as is, but might be a nice refactor to remove if it's not needed...